sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -365,6 +384,22 @@ public void recognizeEntitiesForListWithOptions(HttpClient httpClient, TextAnaly
                 .verifyComplete());
     }
 
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.textanalytics.TestUtils#getTestParameters")
+    public void recognizeEntitiesBatchTooManyDocuments(HttpClient httpClient, TextAnalyticsServiceVersion serviceVersion) {
+        client = getTextAnalyticsAsyncClient(httpClient, serviceVersion);
+        tooManyDocumentsRunner(inputs ->
+            StepVerifier.create(client.recognizeEntitiesBatch(inputs, null, null))
+                .verifyErrorSatisfies(ex -> {
+                    HttpResponseException httpResponseException = (HttpResponseException) ex;
+                    assertEquals(400, httpResponseException.getResponse().getStatusCode());
+                    TextAnalyticsError textAnalyticsError = (TextAnalyticsError) httpResponseException.getValue();
+                    // TODO: TextAnalyticsError has null values for all properties,
```

Comment:
just to clarify, you're going to start asserting the error code and message once you regenerate with the swagger fix?

Reply:
I will only assert the error code. 
Johan's comments in Cognitive Scrum chat:  
""Note: we should not (in the general case) be asserting on error messages. I would not consider localizing error messages to be a breaking change, for example..."" and "Yes. I assume that we don't have client library dependencies on the message itself (it's been done in the past, and it has not ended well)"